superai/data_program/protocol/task_helpers.py

```diff
@@ -0,0 +1,186 @@
+import superai_schema.universal_schema.task_schema_functions as df
+from superai.data_program.combiner.agreement_scores import (
+    agreement_basic,
+    agreement_exclusive_choice,
+    agreement_multiple_choice,
+)
+from superai.data_program.combiner.combiner_functions import (
+    basic_majority,
+    exclusive_choice_majority,
+    multiple_choice_majority,
+)
+from superai.data_program.protocol.task import get_task_type, get_task_value, task
+from superai.data_program.Exceptions import TaskExpired, UnexpectedDataType, UnknownTaskStatus
+
+CM_OFFICE_METRIC = "crowd_manager_office"
+
+
+def resend_task(
+    task_inputs,
+    task_outputs,
+    task_expiry_time,
+    n_resend,
+    task_name,
+    qualifications=None,
+    groups=None,
+    excluded_ids=None,
+    show_reject=False,
+    amount=None,
+    task_price=None,
+):
+    for n_tries in range(n_resend):
+        result = task(
+            input=task_inputs,
+            output=task_outputs,
+            name=task_name,
+            price=task_price,
+            qualifications=qualifications,
+            groups=groups,
+            time_to_expire_secs=task_expiry_time,
+            excluded_ids=excluded_ids,
+            show_reject=show_reject,
+            amount=amount,
+        ).result()
+        if result.status() == "COMPLETED":
+            print("task succeeded")
+            getter = getattr(result.response(), "get", None)
+            if callable(getter) and len(getter("values", [])) > 0:
+                return result
+            else:
+                print("WARNING: completed task, but empty task response.")
+                print("resending task, trial no. ", n_tries + 1)
+                continue
+        elif result.status() in ["EXPIRED", "REJECTED"]:
+            print("resending task, trial no. ", n_tries + 1)
+            continue
+        else:
+            raise UnknownTaskStatus(str(result.status()))
+    raise TaskExpired("No crowd hero responded to task after " + str(n_resend) + "retries.")
+
+
+def resend_task_prioritize_cm_office(
+    task_inputs,
+    task_outputs,
+    qualifications,
+    task_price,
+    task_expiry_time,
+    n_resend,
+    task_name,
+    excluded_ids=None,
+):
+    from canotic.hatchery import turbine_api as tb  # TODO: Remove dependency
```

Comment:
Still depending on `canotic`

Reply:
Yes, this will be removed in 0.1.0 release